wd_capabilities.au3

```diff
@@ -147,17 +128,19 @@ Global $_WD_CAPS__CURRENTIDX = -1
 ; Example .......: No
 ; ===============================================================================================================================
 Func _WD_CapabilitiesStartup()
+	Local Const $sFuncName = "_WD_CapabilitiesStartup"
+
 	$_WD_CAPS__OBJECT = ''
-	ReDim $_WD_CAPS__API[0][$_WD_CAPS__COUNTER]
+	__WD_ConsoleWrite($sFuncName & ': #' & @ScriptLineNumber, $_WD_DEBUG_Full)
 EndFunc   ;==>_WD_CapabilitiesStartup
 
 ; #FUNCTION# ====================================================================================================================
 ; Name ..........: _WD_CapabilitiesAdd
 ; Description ...: Add capablitities to JSON string
 ; Syntax ........: _WD_CapabilitiesAdd($key[, $value1 = ''[, $value2 = '']])
 ; Parameters ....: $key                 - Capability or Match type defined in $_WD_KEYS__*
-;                  $value1              - [optional] a variant value. Default is ''.
-;                  $value2              - [optional] a variant value. Default is ''.
+;                  $value1              - [optional] a variant value. Default is Default.
+;                  $value2              - [optional] a variant value. Default is Default.
```

Comment:
I changed these back to '' in the prior PR, which is inline with the rest of the UDF.

Reply:
Ah... I see.
I will back it too.

Reply:
Take a look here: 
[Default is ''](https://github.com/Danp2/au3WebDriver/pull/285/commits/d578b03e6025e847e0878a9c8b550adf75dacb8c)